test/integration/authoritative-postgres/default_spec.rb

```diff
@@ -21,9 +21,7 @@
   it { should exist }
 end
 
-describe processes('pdns_server-authoritative-server-01-instance') do
-  its ('users') { should eq [default_authoritative_run_user] }
-end
+check_process_name('server-01', default_authoritative_run_user)
```

Comment:
Like a said in the commit message, the partial search in `inspec` has some bugs https://github.com/chef/inspec/issues/1497 and https://github.com/chef/inspec/issues/1867, so just took the code from inspec changed it a bit and let's wait for them to fix thoses bugs.